### PR TITLE
Map Dump 2.0 (accidentallly closed pr)

### PR DESCRIPTION
**Pull request by ScarSkit**
## About The Pull Request

Added 5 maps, two shuttle templates, and improved g corp

## Why It's Good For The Game

More map variety and quantity

## Changelog
:cl:
add: Added etacorp
add: Added thetacorp
add: Added iotacorp
add: Added kappacorp and kappacommand
add: Added lambdacorp
add: Added zetacorp
tweak: tweaked g corp
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Capitalize letters.

File: _maps/iotacorp.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"version": 1,
+	"map_name": "Facility i-423",
```
Comment (Kitsunemitsu) [inline _maps/iotacorp.json:3]: I-423

Comment (ScarSkit) [reply _maps/iotacorp.json:3]: ahh I see thanks

**Comment by Kitsunemitsu** [comment]
This is a massive PR, with many issues, but when you are satisfied with the shape of each map we can continue and I can help you polish up. 
Next time I'd prefer atomization but we can talk about that later

**Pull request closed without merging**